parm/config/gfs/config.resources

```diff
@@ -289,6 +289,14 @@ case ${step} in
     export npe_node_snowanl=$(( npe_node_max / nth_snowanl ))
     ;;
 
+  "prepobsaero")
+    export wtime_prepobsaero="00:30:00"
+    export npe_prepobsaero=1
+    export nth_prepobsaero=1
+    export npe_node_prepobsaero=1
+    export memory_prepobsaero="96GB"
```

Comment:
Does this prep job really need all available memory?

Reply:
Probably not, but it is somewhat intensive, but this is probably an order of magnitude too large, I can test next week with less memory